src/sage/matrix/matrix0.pyx

```diff
@@ -5813,10 +5813,11 @@ cdef class Matrix(sage.structure.element.Matrix):
             Traceback (most recent call last):
             ...
             ArithmeticError: self must be a square matrix
+
             sage: matrix(RR, 1, 1, [2]).inverse_of_unit()                               # needs sage.libs.singular
             Traceback (most recent call last):
             ...
-            NotImplementedError: Lifting of multivariate polynomials over non-fields is not implemented.
+            ArithmeticError: non-invertible matrix
 
```

Comment:
I get 
```
sage: M = matrix(RR, 1, 1, [2])
sage: M.is_invertible()
True
sage: M.inverse_of_unit()
[0.500000000000000]
```
as of sage 10.2.beta6.

Reply:
With the definition of RR as in the tests here? I agree it is a confusing set up

Reply:
Ah, I missed the definitions that come before.

OK.